src/scss/mixins/variables/_animation.scss

```diff
@@ -0,0 +1,10 @@
+$animation-time: 250ms !default;
+
+@mixin componentAnimationVars() {
+  // Component Animation Variables
```

Comment:
Need to make these CSS comments or it won't propagate to output CSS. Same with the others.